protocol/urls/urls.go

```diff
@@ -293,3 +293,11 @@ func GetLinkPreviewData(link string) (previewData LinkPreviewData, err error) {
 		return previewData, fmt.Errorf("link %s isn't whitelisted. Hostname - %s", link, u.Hostname())
 	}
 }
+
```

Comment:
Worth mentioning that I created a different package `linkpreview` in `protocol/` for dealing specifically with link previews and  the idea was to delete the `urls` package entirely since it doesn't fulfill clients' needs anymore.

I think the PR can be merged using the `urls` package, but I think we should refactor soon(ish) to use the `Unfurler` interface defined in `protocol/linkpreview` so that we don't grow the package that was supposed to be deleted (when the new link preview implementation is more complete featurewise).

**OR forget what I just said above**, and we can use/think of a completely different package and interface to deal with "domain" data (community, channel, user) because these can be seen as significantly different from unfurleable data from websites. So maybe using the package `urls` is okay. Food for thought :)